client/src/app/site/polls/components/base-poll-detail.component.ts

```diff
@@ -116,19 +116,17 @@ export abstract class BasePollDetailComponent<V extends ViewBasePoll> extends Ba
     }
 
     public async deletePoll(): Promise<void> {
-        const title = 'Delete poll';
-        const text = 'Do you really want to delete the selected poll?';
+        const title = 'Are you sure you want to delete this vote?';
```

Comment:
Sometimes you are using translate here and sometime you are not. Stay consistent

Reply:
fixed.